variables.tf

```
@@ -5,9 +5,18 @@ variable "ec2_instance_connect" {
 }
 
 variable "download_url" {
+  type        = map
+  default = {
+    "1.19.3" = "https://piston-data.mojang.com/v1/objects/c9df48efed58511cdd0213c56b9013a7b5c9ac1f/server.jar"
+    "1.19.2" = "next url"
```

I know that `1.19.2` is available here:
```suggestion
    "1.19.2" = "https://piston-data.mojang.com/v1/objects/f69c284232d7c7580bd89a5a4931c3581eae1378/server.jar"
```
But the rest are a mystery :)

---

Hi 
There are many unofficial sources on the google. I am not able to judge to what extent they are reliable. To be honest I never played minecraft and I am not intended to do so. I do not understand what the benefit is of using older versions.
Have you seen this one: https://hub.docker.com/r/itzg/minecraft-server ?
Aslo an open question where the user's data is going to be stored, for example what is in their bag.

B.